changelogs/unreleased/gh-7954-sort-spaces-in-snapshot-by-id.md

```diff
@@ -0,0 +1,3 @@
+## feature/box
+
+* Now all spaces in a snapshot file are sorted by their id (gh-7954).
```

Comment:
The doc doesn't say that spaces with id < 256 are stored after system spaces. Also, we should probably update the doc to mention that spaces are guaranteed to be sorted by id only in Tarantool 3.0 or later.

Reply:
Changed the changelog and TarantoolBot request.